r/NAMESPACE

```diff
@@ -63,7 +63,6 @@ S3method(as_nanoarrow_schema,nanoarrow_schema)
 S3method(as_nanoarrow_schema,nanoarrow_vctr)
 S3method(c,nanoarrow_vctr)
 S3method(convert_array,default)
-S3method(convert_array,double)
```

Comment:
I had initially thought this might be breaking but I see the way you have things hooked up here `convert_array(..., to = double())` works just like before. 👍